components/policy/resources/templates/policy_definitions/BraveSoftware/BraveTrackingQueryParametersFilteringDisabled.yaml

```diff
@@ -0,0 +1,36 @@
+caption: Disable tracking query parameters filtering
+default: null
+desc: |-
+  Tracking query parameters filtering allows Brave to automatically strip known tracking query parameters from URLs.
+
+            If this policy is set to true, tracking query parameters filtering will always be disabled.
+
+            If this policy is set to false, tracking query parameters filtering will always be enabled.
```

Comment:
Is this true? Or will it just be up to users? It's a bit confusing the way it's worded given the name of the policy. Is this the pattern we've been following?

Reply:
@ShivanKaul, renamed and changed policy description